scripts/new-solr-updater.py

```diff
@@ -36,6 +36,8 @@ def parse_arguments():
     parser.add_argument('-c', '--config')
     parser.add_argument('--debugger', action="store_true", help="Wait for a debugger to attach before beginning.")
     parser.add_argument('--state-file', default="solr-update.state")
+    parser.add_argument('--exclude-edits',
```

Comment:
Would it clearer to call it `--exclude-edits-containing` ? the help text referring to `the string` doesn't make it much clearer for me

Reply:
I think that might be a little too verbose; maybe changing the helptext to `Don't index edits containing this argument anywhere`?

Reply:
Fixed